portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
+
+
 public class DataServlet extends HttpServlet {
+  
+  private ArrayList<String> list;
+
+  public DataServlet(){ 
+    list = new ArrayList<String>();
+      list.add("Fruit");
+      list.add("Soap");
+      list.add("Cereal");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("Hello Bria!");
```

Comment:
Do you still need this?

Reply:
They want us to build off of the previous walkthroughs, so I've been keeping them but I can delete them if that is what you think is best.

Reply:
Yeah, they are overridden by line 42 and 43. So no point of keeping them.